experimental/plugins/postgres_storage/src/postgres_storage.rs

```diff
@@ -480,6 +483,7 @@ struct MultiWalletMultiTableStrategy {}
 impl WalletStrategy for DatabasePerWalletStrategy {
     // initialize storage based on wallet storage strategy
     fn init_storage(&self, _config: &PostgresConfig, _credentials: &PostgresCredentials) -> Result<(), WalletStorageError> {
+        env_logger::try_init();
```

Comment:
warning: unused `std::result::Result` that must be used
   --> src/postgres_storage.rs:486:9
    |
486 |         env_logger::try_init();
    |         ^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: #[warn(unused_must_use)] on by default
    = note: this `Result` may be an `Err` variant, which should be handled

warning: unused `std::result::Result` that must be used
   --> src/postgres_storage.rs:652:9
    |
652 |         env_logger::try_init();
    |         ^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: this `Result` may be an `Err` variant, which should be handled